src/commands/qmis.ts

```diff
@@ -88,6 +96,18 @@ export default class QMIS extends Command {
         await DELETE.qmis(id);
         this.log('QMI deletion successful.');
       } else { this.log('Typed response doesn\'t match QMI IP address, aborting deletion.'); }
+    } else if (flags.start || flags.stop) {
+      if (flags.keypath) throw new Error(`Cannot supply a --keypath when powering on a QMI.`);
+      if (!flags.id) throw new Error(`Must supply an --id when deleting a QMI.`);
+
+      const id = parsePositiveInteger(flags.id);
+      if (!id) { throw new Error(`Must supply a positive integer ID when powering on/off a QMI.`); }
+
+      try {
+        const qmiURLAction = flags.start ? 'start' : 'stop';
```

Comment:
V small detail, but if a user supplies both `--start` and `--stop` this will default to `--start`. Like the other key combo compatibility checks we do, you could add a quick check above that only one is supplied, not both